app/web/src/utils/typescriptLinter.ts

```diff
@@ -94,9 +153,6 @@ export const createTypescriptSource = async (
       docString.trim().length === 0 ? fallbackCode : docString,
     );
 
-    // TODO: use getQuickInfoAtPosition to display the types definitions on hover
```

Comment:
Hahahah I can't believe I wrote this and then forgot it forever, thank you for fixing it! This PR is awesome!